composer.json

```diff
@@ -29,11 +29,11 @@
     "require-dev": {
         "roave/security-advisories": "dev-latest",
         "codeception/codeception": "4.1.*",
-        "squizlabs/php_codesniffer": "^3.6.0",
-        "overtrue/phplint": "^3.0.0",
-        "friendsofphp/php-cs-fixer": "^3.0.0",
+        "squizlabs/php_codesniffer": "^3.6",
+        "overtrue/phplint": "^3.0",
+        "friendsofphp/php-cs-fixer": "^3.0",
         "phpunit/phpunit": ">=9.0",
```

Comment:
PHPUnit 10 is not supported, so this constraint should be `^9.5`.

Reply:
Ah Thanks!